appointment/upload-documents/uploadDocuments.php

```diff
@@ -73,21 +73,142 @@ class="submit dcf-btn dcf-btn-primary"
+									ng-model="consentData.reviewConsent" 
+									uib-btn-radio="1" 
+									type="radio"
+									required>Yes
+								</label>
+								<label class="dcf-btn dcf-btn-secondary" 
+									name="consentToReview" 
+									ng-model="consentData.reviewConsent" 
+									uib-btn-radio="0" 
+									type="radio"
+									required>No
+								</label>
+							</div>
+						</div>
+						<div ng-show="consentForm.$submitted">
+							<label class="error" ng-show="consentForm.consentToReview.$error.required">This field is required.</label>
+						</div>
+					</li>
 
-				<div class="dcf-input-checkbox">
-					<input id="agree-to-virtual-preparation-checkbox" type="checkbox" ng-model="agreeToVirtualPreparationCheckbox.checked" value="false">
-					<label for="agree-to-virtual-preparation-checkbox">I agree to have my tax return prepared virtually</label>
-				</div>
+					<p class="dcf-bold">Virtual Consent Disclosure:</p>
+					<p>If you agree to have your tax return prepared and your tax documents handled in the above manner,
+						your signature and/or agreement is required on this document. Signing this document means that
+						you are agreeing to the procedures stated above for preparing a tax return for you. (If this
+						is a Married Filing Joint return both spouses must sign and date this document.) If you chose
+						not to sign this form, we may not be able to prepare your tax return using this process. Since
+						we are preparing your tax return virtually, we have to secure your consent agreeing to this 
+						process. If you consent to use these non-IRS virtual systems to disclose or use your tax return
+						information, Federal law may not protect your tax return information from further use or distribution
+						in the event these systems are hacked or breached without our knowledge. If you agree to the disclosure
+						of your tax return information, your consent is valid for the amount of time that you specify. If
+						you do not specify the duration of your consent, your consent is valid for one year from the date
+						of signature. If you believe your tax return information has been disclosed or used improperly in a
+						manner unauthorized by law or without your permission, you may contact the Treasury Inspector General
+						for Tax Administration (TIGTA) by telephone at 1-800-366-4484, or by e-mail at complaints@tigta.treas.gov .
+						While the IRS is responsible for providing oversight requirements to Volunteer Income Tax Assistance (VITA) 
+						and Tax Counseling for the Elderly (TCE) programs, these sites are operated by IRS sponsored partners who
+						manage IRS site operations requirements and volunteer ethical standards. In addition, the locations of these 
+						ites may not be in or on federal Property. </p>
+					<li class="form-radio" id="consentVirtual">
+						<label class="form-required">I am agreeing to use this site's Virtual VITA/TCE Process</label>
```

Comment:
There isn't anything for me to include in the label for the first yes/no question, so I don't know if I should take this out of the label and just leave it as standard text for consistency or what.

The first question (consenting to participate in random IRS quality assurance review of the site) is optional in that they can say yes or no, but for this one (consenting to virtual preparation) they have to say yes. I have made both "required" on the form, but should I make them `class="form-required"` so that they have the little "required" label? Might make clients think they have to say yes.

I mention it elsewhere, but if I take off the controller validation in the ng-disable for the submit button and rely just on form validation, then if they try to submit without filling out the form I can show the `class="error"` label and say something along the lines of "You must answer this question but can say yes or no."

Reply:
I would add required to the actual required toggle. And I think in the interest of transparency, I'd probably add some small text beneath the "IRS" one saying "This is optional, but helps ensure Lincoln VITA is meeting the IRS quality guidelines for tax preparation" or something like that--but maybe pitch this question to Tracy.

Reply:
Okay--label on the first question is back on. I think it looks kinda awkward, but it's probably not that big of a deal.